app/policies/application_policy.rb

```diff
@@ -11,7 +11,7 @@ def show?
   end
 
   def index?
-    @user.admin?
+    @user.admin? if @user
```

Comment:
Or move to `MembershipApplicationPolicy` now that there is more views in the system. That is the index view we want to restrict.